lib/messages.js

```diff
@@ -162,6 +162,7 @@ exports.noisePayload = {
     if (m.udx) udxInfo.preencode(state, m.udx)
     if (m.secretStream) secretStreamInfo.preencode(state, m.secretStream)
     if (m.relayThrough) relayThroughInfo.preencode(state, m.relayThrough)
+    if (m.timestamp >= 0) c.uint.preencode(state, m.timestamp)
```

Comment:
just > 0

Reply:
IDK, being able to send a 0 timestamp could be useful... 😆

Reply:
```suggestion
    if (m.timestamp > 0) c.uint.preencode(state, m.timestamp)
```